daos/movie_info.go

```diff
@@ -0,0 +1,61 @@
+package daos
+
+import (
+	"database/sql"
+
+	"github.com/ashwinspg/explore-golang/dtos"
+)
+
+//MovieInfo - DAO
+type MovieInfo struct {
+	db *sql.DB
+}
+
+//NewMovieInfo - instance creation
+func NewMovieInfo(db *sql.DB) *MovieInfo {
+	return &MovieInfo{db}
+}
+
+//Save - add new MovieInfo data
+func (movieInfoDAO *MovieInfo) Save(movieInfoDTO dtos.MovieInfo) error {
+	query := `INSERT INTO "` + dtos.MovieInfoTable + `"("movie_uuid", "info") VALUES($1, $2)`
+
+	statement, err := movieInfoDAO.db.Prepare(query)
+
+	if err != nil {
+		return err
+	}
+
+	defer statement.Close()
+
+	_, err = statement.Exec(movieInfoDTO.Movie_UUID, movieInfoDTO.Info)
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+//FindByID - find movie_info through id
+func (movieInfoDAO *MovieInfo) FindByID(uuid string) (dtos.MovieInfo, error) {
+	query := `SELECT * FROM ` + dtos.MovieInfoTable + ` WHERE "movie_uuid" = $1`
```

Comment:
Note:
For simple SQL statements such as these, it is OK to pass the uuid in QueryRow
But in case of complex queries, using inline query arguments will help in making it readable.
```suggestion
	query := `SELECT * FROM ` + dtos.MovieInfoTable + ` WHERE movie_uuid =` + uuid  + `;`
```